spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -71,9 +71,16 @@ def assert_authentication
   def assert_specific_container
     @container = Container.find_by_name("heapster")
     @container.should have_attributes(
-      # :ems_ref       => "a7566742-e73f-11e4-b613-001a4a5f4a02_heapster_kubernetes/heapster:v0.9",
-      :name          => "heapster",
-      :restart_count => 2,
+      # :ems_ref        => "a7566742-e73f-11e4-b613-001a4a5f4a02_heapster_kubernetes/heapster:v0.9",
+      :name             => "heapster",
+      :restart_count    => 2,
+      :state            => "running",
+      :started_at       => "2015-07-29T15:49:04Z",
+      :last_state       => "terminated",
+      :last_exit_code   => 1,
+      :last_started_at   => "2015-07-29T15:48:42Z",
+      :last_finished_at => "2015-07-29T15:48:42Z",
```

Comment:
> These will be a mess next time we'll do a new recording :sob:

@simon3z @oschreib These should be commented out like ```ems_ref```.  The overhead of changes for every recording is too much.